test/recipes/90-test_store.t

```diff
@@ -89,43 +100,71 @@ indir "store_$$" => sub {
 
         foreach (@noexist_files) {
             my $file = srctop_file($_);
-            ok(!run(app(["openssl", "storeutl", $file])));
-            ok(!run(app(["openssl", "storeutl", to_abs_file($file)])));
-            ok(!run(app(["openssl", "storeutl", to_abs_file_uri($file)])));
+        SKIP: {
+                ok(!run(app(["openssl", "storeutl", $file])));
+                ok(!run(app(["openssl", "storeutl", to_abs_file($file)])));
+
+                skip "No test of URI form of $file on mingw", 1 if $msys;
```

Comment:
"on msys" or "under msys" perhaps? Multiple occurrences. Though if check is modified for combination of $^O and config('target'), then it would be appropriate to keep reference to mingw, but maybe as [little tongue-in-cheek] "for mingw".